recon_surf/recon-surf.sh

```diff
@@ -556,44 +548,22 @@ fi
 
 # ============================= TALAIRACH ==============================================
 
-if [ "$long" == "1" ] ; then
-  #TODO: move this processing into talairach-reg.sh for consistency.
```

Comment:
Why was this necessary? For long we do not run any talairach, so instead we copy things. this should not be part of a talairach-reg script as it complicates things and makes debugging more difficult.  Happy to be convinced otherwise, but I see no good reason for this.

Reply:
My logic was that talairach registration can be called from either segmentation or surface and this way it always gets created in the right way, i.e. if people specify `--tal_reg`, it would before do the full talairach registration here: https://github.com/Deep-MI/FastSurfer/blob/4de8b3af5866f891561b6cef4ae057412dc968e3/run_fastsurfer.sh#L833

So my logic was that if the talairach registration script was clear, and would do the right thing for longitudinal (i.e. copy and have a `--long` flag) that would actually be easier to maintain and do... talairach registration stuff is always in that script and that script knows how to do it for longitudinal.

If you disagree, maybe we should change this back.

Reply:
The alternative would have been to add an extra check in run_fastsurfer, that ALWAYS disables --tal_reg for --long processing. This would also have addressed this specific problem.
My idea was to put things concerning talairach registration into that script and solving it in there. (Basically just moving this whole if-else-clause into the script (also some of the steps like adding the talreg into nu.mgz were duplicated).